test/kwtool_web/views/api/v1/error_view_test.exs

```diff
@@ -0,0 +1,96 @@
+defmodule KwtoolWeb.Api.V1.ErrorViewTest do
+  use KwtoolWeb.ConnCase, async: true
+
+  # Bring render/3 and render_to_string/3 for testing custom views
+  import Phoenix.View
```

Comment:
Maybe we can use ViewCase instead

Reply:
Yes, i updated in https://github.com/byhbt/kwtool/pull/45/commits/305904ac02be117c7e1f55d5ef9479984d526519